spec/mdn/mdn.mdown

```diff
@@ -0,0 +1,278 @@
+
+MDN are defined in [@!RFC8098] and are used as "read receipts", "acknowledgements", or "receipt notifications".
+
+A client can have to deal with MDN in different ways:
+
+1. When receiving an email, an MDN can be sent to the sender. This specification defines an EmailSubmission/sendMDN method to cover this case.
+2. When sending an email, an MDN can be requested. This must be done with the help of a header, and is already specified by [@!RFC8098] and can already be handled by [@!I-D.ietf-jmap-mail] this way.
+3. When receiving an MDN, the MDN could be related to an existing sent mail. This is already covered by [@!I-D.ietf-jmap-mail] in the EmailSubmission object. Client could want to display detailed information about a received MDN. This specification defines a EmailSubmission/parseMDN method to cover this case.
+
+## Notational conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [@!RFC2119] [@?RFC8174] when, and only when, they appear in all capitals, as shown here.
+
+Type signatures, examples and property descriptions in this document follow the conventions established in section 1.1 of [@!I-D.ietf-jmap-core]. Data types defined in the core specification are also used in this document.
+
+Servers MUST support all properties specified for the new data types defined in this document.
+
+## Terminology
+
+The same terminology is used in this document as in the core JMAP specification.
+
+## Addition to the capabilities object
+
+The capabilities object is returned as part of the standard JMAP Session object; see the JMAP spec. Servers supporting *this* specification MUST add a property called `urn:ietf:params:jmap:mdn` to the capabilities object.
+
+# MDN
+
+An **MDN** object has the following properties:
+
+- **forEmailId**: `String`
+  Email Id of the received email this MDN is relative to.
+- **subject**: `String|null`
+  Subject used as `Subject` header for this MDN.
+- **textBody**: `String|null`
+  Human readable part of the MDN, as plain text.
+- **reportingUA**: `String|null`
+  Name of the MUA creating this MDN. It is used to build the MDN Report part of the MDN.
+- **disposition**: `Disposition`
+  Object containing the diverse MDN disposition options.
+- **mdnGateway**: `String|null` (server-set)
+  Name of the gateway or MTA that translated a foreign (non-Internet) message disposition notification into this MDN.
+- **originalRecipient**: `String|null` (server-set)
+  Original recipient address as specified by the sender of the message for which the MDN is being issued.
+- **finalRecipient**: `String` (server-set)
+  Recipient for which the MDN is being issued.
+- **originalMessageID**: `String|null` (server-set)
```

Comment:
is it of any use? I mean, we already have a mandatory forEmailId field that allows to find the original Email.

Reply:
yes, when parsing an MDN there can be cases where you have the originalMessageID field but no matching email id